### PR TITLE
Update contact section

### PR DESCRIPTION
**Pull request by raamSoftwire**
### Context

Update the contact details section to be inline with other copy changes.

### Changes proposed in this pull request

- changed "Contact details" to "Contact this training provider"
- updated opening hours from "8am to 8pm" to "8.30am to 5pm"
- changed the opening hours string from "Monday to Friday, 8.30am to 5pm (except public holidays)" to "8am to 8pm, Monday to Friday"
- updated the copy in the "advice about teaching section"
- updated the details in the footer of the page

#### Before

image
image

#### After

image
image

### Trello card

https://trello.com/c/67mWQbAv/3057-changing-contact-details-section-on-provider-page-in-find

### Checklist

- [x] Rebased master
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

**Comment by Tomcowan1** [comment]
Hi @raamSoftwire thanks for taking a look at this (and sorry all of the changes didn't come through on the Google doc). Couple of edits to make:

- Remove subtitle _For advice about teaching_ (_Support and advice_ is the only title)

- Remove the content in the _Support and advice_ section and replace with:

For support and advice, you can speak to a Get Into Teaching adviser for free. They’re all experienced teachers who can help you to prepare your application, book school experience, and access exclusive teaching events.

You can also call Get Into Teaching free of charge on 0800 389 2501, or speak to an adviser using their online chat service Monday to Friday, 8.30am to 5pm (except public holidays).

- Opening hours should be expressed as: Monday-Friday between 8.30am and 5pm.

**Comment by raamSoftwire** [comment]
@Tomcowan1 I've just posted an updated screenshot

**Comment by Tomcowan1** [comment]
Looks good - thanks @raamSoftwire !

**Review by Tomcowan1 — APPROVED** [review]

File: app/views/courses/_advice.html.erb [hunk 1]
```diff
@@ -1,10 +1,21 @@
 <div class="govuk-!-margin-bottom-8">
   <h3 class="govuk-heading-l" id="section-advice">Support and advice</h3>
-  <p class="govuk-body">For questions about this course you should contact the training provider using <%= govuk_link_to 'the contact details above', '#contact_section' %>.</p>
 
-  <h4 class="govuk-heading-m">Get support and advice about teaching</h4>
-  <p class="govuk-body">You can speak to a <%= govuk_link_to t('service_name.get_into_teaching'), t('get_into_teaching.url_get_an_advisor') %> adviser for free. They’re all experienced teachers who can help you prepare your application, book school experience, and access exclusive teaching events.</p>
-  <p class="govuk-body">You can also call <%= t('service_name.get_into_teaching') %> free of charge on <%= t('get_into_teaching.tel') %>, or speak to an adviser using their <%= govuk_link_to 'online chat service', t('get_into_teaching.url_online_chat') %> <%= t('get_into_teaching.opening_times') %>.</p>
+<!--  <p class="govuk-body">Register with <%#= govuk_link_to t('service_name.get_into_teaching'), t('get_into_teaching.url_get_an_advisor') %>,-->
```
Comment (davidgisbey) [inline app/views/courses/_advice.html.erb:4]: think this needs to be uncommented.

Comment (raamSoftwire) [reply app/views/courses/_advice.html.erb]: Just spotted that, it needed to come out. I'd left it in to steal all the i18n strings. It's gone now

Comment (davidgisbey) [reply app/views/courses/_advice.html.erb]: sorry outdated now!

**Review by davidgisbey — APPROVED** [review]

**Pull request merged**